tests/e2e/api/account.test.ts

```diff
@@ -266,15 +257,15 @@ describe("account api", () => {
       await aptos.fundAccount({ accountAddress: account.accountAddress, amount: 1_000_000_000 });
 
       // account that holds the new key
-      const rotateToPrivateKey = Ed25519PrivateKey.generate();
+      const rotateToSigner = Account.generate({ scheme: SigningSchemeInput.Ed25519, legacy: true });
```

Comment:
it is not really needed to specify `{ scheme: SigningSchemeInput.Ed25519, legacy: true }` right? `Account.generate()` defaults to it

Reply:
Yeah that's correct. I actually rolled back that part as well